perf/sds-tests/citadel-1/README.md

```diff
@@ -0,0 +1,67 @@
+## Test description
```

Comment:
It may worth to call out whether controlPlaneSecurity is enabled or not and it is great if we can cover both cases.

Reply:
Added a description of whether controlPlaneSecurity is enabled or not, and the instruction to turn on  controlPlaneSecurity.